CSL JSON.js

```diff
@@ -56,7 +56,10 @@ function detectImport() {
 }
 
 function doImport() {
-	if(!parsedData) parseInput();
+	//We do call parseInput always because otherwise multiple async translator
+	//calls might interfer together.However, it may be that parsedData is
+	//already set from detectImport.
+	parseInput(); 
```

Comment:
In that case, avoid the global altogether and have `parseInput` return `parsedData`.

FYI, this is the same issue as https://github.com/zotero/translators/commit/bc4adc0d1e026657d9624df118eb862bc4290818